packages/storybook/stories/va-breadcrumbs-uswds.stories.jsx

```diff
@@ -16,9 +16,11 @@ export default {
 
 const Template = ({ label, 'disable-analytics': disableAnalytics, uswds }) => (
   <va-breadcrumbs uswds label={label} disable-analytics={disableAnalytics}>
-    <a href="#home">Home</a>
-    <a href="#one">Level one</a>
-    <a href="#two">Level two</a>
+    <span slot="list">
```

Comment:
Wouldn't making this markup change mean that all of the existing v1 `va-breadcrumbs` implementations will need to be changed? This would be a breaking change then and we probably want to avoid it.